osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -176,6 +176,8 @@ private class OsuHitObjectLifetimeEntry : HitObjectLifetimeEntry
             public OsuHitObjectLifetimeEntry(HitObject hitObject)
                 : base(hitObject)
             {
+                // Arbitrary lifetime end to prevent past objects in idle states remaining alive in non-frame-stable contexts.
```

Comment:
Does this comment still make sense? I don't believe the `+1000` is stopping objects from remaining alive, as much as giving them a chance to be judged or missed?

Reply:
It looks like the `+1000` isn't necessary. DHOs set `LifetimeEnd = double.MaxValue` on state update (i.e. when they come alive), and a more appropriate lifetime is set from there. Will remove.